[2023-07-19T20:41:15.410+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.extract_data manual__2023-07-19T20:41:10.078951+00:00 [queued]>
[2023-07-19T20:41:15.683+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.extract_data manual__2023-07-19T20:41:10.078951+00:00 [queued]>
[2023-07-19T20:41:15.684+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-19T20:41:15.716+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): extract_data> on 2023-07-19 20:41:10.078951+00:00
[2023-07-19T20:41:15.725+0000] {standard_task_runner.py:57} INFO - Started process 269 to run task
[2023-07-19T20:41:15.730+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'extract_data', 'manual__2023-07-19T20:41:10.078951+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmpbu53007x']
[2023-07-19T20:41:15.739+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask extract_data
[2023-07-19T20:41:15.831+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.extract_data manual__2023-07-19T20:41:10.078951+00:00 [running]> on host 90317fb99301
[2023-07-19T20:41:15.949+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fvinciguerra' AIRFLOW_CTX_DAG_ID='ETL_stock_data_google_DAG' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-19T20:41:10.078951+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-19T20:41:10.078951+00:00'
[2023-07-19T20:41:15.952+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-19T20:41:15.956+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/pyscripts/Extract_stock_data_google.py']
[2023-07-19T20:41:15.968+0000] {subprocess.py:86} INFO - Output:
[2023-07-19T20:41:19.942+0000] {subprocess.py:93} INFO - 2023-07-19 20:41:19,942 - INFO - Iniciando extracción de datos...
[2023-07-19T20:41:19.943+0000] {subprocess.py:93} INFO - 2023-07-19 20:41:19,943 - INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 17] File exists: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2023-07-19T20:41:20.868+0000] {subprocess.py:93} INFO - 2023-07-19 20:41:20,868 - INFO - Extracción de datos completada.
[2023-07-19T20:41:20.896+0000] {subprocess.py:93} INFO - 2023-07-19 20:41:20,896 - INFO - Datos extraídos guardados exitosamente.
[2023-07-19T20:41:21.025+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-19T20:41:21.060+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=ETL_stock_data_google_DAG, task_id=extract_data, execution_date=20230719T204110, start_date=20230719T204115, end_date=20230719T204121
[2023-07-19T20:41:21.107+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-19T20:41:21.124+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
